UPGRADE-4.0.md

```diff
@@ -15,10 +15,16 @@ Form
  * The `choices_as_values` option of the `ChoiceType` has been removed.
 
  * Support for data objects that implements both `Traversable` and
-   `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed
+   `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed.
 
- * The following form types registered as services has been removed; use
-   their fully-qualified class name instead:
+FrameworkBundle
+---------------
+
+ * Support for absolute template paths has been removed from the template name
+   parser.
```

Comment:
I am not aware of this change. What should be the sentence though:
`Support for absolute template paths has been removed` ?